crates/typst-library/src/math/matrix.rs

```diff
@@ -31,13 +31,28 @@ pub struct VecElem {
     /// The elements of the vector.
     #[variadic]
     pub children: Vec<Content>,
+
+    /// The gap between elements.
+    ///
+    /// ```example
+    /// #set math.vec(gap: 3em)
+    /// $ vec(1, 2) $
+    /// ```
+    #[resolve]
+    #[fold]
+    pub gap: Option<Rel<Length>>,
```

Comment:
All of these shouldn't be `Option`. Instead, they should use the `#[default(..)]` macro to set an explicit default value. Otherwise, a user can write `#set math.vec(gap: none)` to get the default, which would be strange.